packages/components/src/components/hds/form/super-select/multiple/field.hbs

```diff
@@ -65,7 +63,7 @@
       @triggerComponent={{@triggerComponent}}
       @triggerId={{F.id}}
       @triggerRole={{@triggerRole}}
-      @typeAheadMatcher={{@typeAheadMatcher}}
+      @typeAheadOptionMatcher={{@typeAheadOptionMatcher}}
```

Comment:
Out of curiosity, why did this get renamed?

Reply:
The argument is [wrongly documented as `@typeAheadMatcher`](https://ember-power-select.com/docs/api-reference). [In practice, it is `@typeAheadOptionMatcher`](https://github.com/cibernox/ember-power-select/blob/master/ember-power-select/src/components/power-select.ts#L117). I'll raise this with the author to fix the docs.